database/seeds/AuthTableSeeder.php

```diff
@@ -36,6 +36,7 @@ public function run()
         $this->call(UserTableSeeder::class);
         $this->call(PermissionRoleTableSeeder::class);
         $this->call(UserRoleTableSeeder::class);
+        $this->call(ParkingTableSeeder::class);
```

Comment:
Just discovered that on running php artisan db:seed ,the ParkingTableSeeder Class isn't recognised, maybe it has to be moved from database/seeds/ to database/seeds/Auth/